src/toast/ops/sim_tod_atm.py

```diff
@@ -152,7 +152,12 @@ class SimAtmosphere(Operator):
         help="Maximum wind drift before discarding the volume and creating a new one",
     )
 
-    freq = Quantity(None, allow_none=True, help="Observing frequency")
+    sample_rate = Quantity(
+        None,
+        allow_none=True,
+        help="Rate at which to sample atmospheric TOD before interpolation.  "
+        "Default is no interpolation.",
+    )
```

Comment:
I like specifying the interpolation in this way (as a sampling rate).  It feels more intuitive.